clients/admin-ui/src/pages/classify-systems/index.tsx

```diff
@@ -1,26 +1,55 @@
 import { Heading, Spinner, Stack, Text } from "@fidesui/react";
 import type { NextPage } from "next";
 import { useRouter } from "next/router";
-import React, { useEffect } from "react";
+import { useEffect } from "react";
+import { useDispatch } from "react-redux";
 
+import { useAppSelector } from "~/app/hooks";
 import Layout from "~/features/common/Layout";
+import {
+  selectSystemsForReview,
+  setSystemsForReview,
+} from "~/features/config-wizard/config-wizard.slice";
 import {
   useGetAllClassifyInstancesQuery,
   useGetHealthQuery,
 } from "~/features/plus/plus.slice";
 import { useGetAllSystemsQuery } from "~/features/system";
 import ClassifySystemsTable from "~/features/system/ClassifySystemsTable";
-import { ClassificationStatus } from "~/types/api";
+import { ClassificationStatus, GenerateTypes } from "~/types/api";
 
 const ClassifySystems: NextPage = () => {
   const router = useRouter();
+  const dispatch = useDispatch();
   const { isSuccess: hasPlus, isLoading: isLoadingPlus } = useGetHealthQuery();
-  const { isLoading: isLoadingSystems, data: systems } = useGetAllSystemsQuery(
-    undefined,
-    { skip: !hasPlus }
-  );
+  /**
+   * TODO: fides#1744
+   * Because there is currently no way to associate a scan with its classification,
+   * we attempt to get the classification results for only the systems in review.
+   * However, because this is a page that is waiting for results and so will likely
+   * be refreshed, we can't necessarily rely on knowing the systems we put into review
```

Comment:
Maybe "we can't necessarily rely on knowing _which_ systems were put into review."

Though I think we actually don't have to worry about this case because the redux state is persisted to local storage. The config wizard state is only cleared with an [explicit cancel](https://github.com/ethyca/fides/blob/ed6bc3b59ea1c5af482db03b3cf2c4b9c64e0d5a/clients/admin-ui/src/features/config-wizard/ConfigWizardWalkthrough.tsx#L38). If this case came up, something else is going wrong.

Reply:
mm that's true. I was worried about somebody going directly to the `/classify-systems` URL without having gone through the config wizard first. but maybe in that case we should just render an empty state since scan wouldn't have happened yet.